t_stress/test_stress.py

```diff
@@ -858,20 +919,45 @@ class TestCtrlFrameFlood(tester.TempestaTest):
     """
     }
 
+    stop_flag = False
+
     def setUp(self):
         self.enable_memleak_check()
         super().setUp()
 
+    def __flood(self):
+        client = self.get_client("ctrl_frames_flood")
+        client.start()
+        self.wait_while_busy(client, timeout=self.timeout)
+        client.stop()
+        self.stop_flag = True
+
     @dmesg.limited_rate_on_tempesta_node
     def test(self):
-        client = self.get_client("ctrl_frames_flood")
+        self.start_all_services(client=False)
         tempesta = self.get_tempesta()
 
-        self.start_all_servers()
-        self.start_tempesta()
-        self.start_all_clients()
-        self.wait_while_busy(client, timeout=120)
-        client.stop()
+        client = self.get_client("deproxy")
+        client.start()
+        client.update_initial_settings()
+        client.send_bytes(client.h2_connection.data_to_send())
+        client.h2_connection.clear_outbound_data_buffer()
+
+        self.assertTrue(
+            client.wait_for_ack_settings(),
+            "Tempesta foes not returns SETTINGS frame with ACK flag.",
+        )
+
+        request = client.create_request(method="GET", headers=[])
+
+        t = threading.Thread(target=self.__flood)
+        t.start()
+
+        for _ in range(1, 20):
+            client.send_request(request, "200")
+        while not self.stop_flag:
+            pass
+        self.stop_flag = False
```

Comment:
Why do you use `Thread` here? Our external clients run as `Process` from `multiprocessing`. 
```suggestion
        flood_client = self.get_client("ctrl_frames_flood")
        flood_client.start()

        for _ in range(1, 20):
            client.send_request(request, "200")

        self.wait_while_busy(flood_client, timeout=self.timeout)
        flood_client.stop()
```